packages/salesforcedx-vscode-apex-debugger/package.json

```
@@ -193,7 +193,7 @@
               },
               "salesforceProject": {
                 "type": "string",
-                "description": "%sfdx_project_text%",
+                "description": "%salesforce__text%",
```

Looks like the word "project" got missing here.

```suggestion
                "description": "%salesforce_project_text%",
```